packages/fxa-settings/src/styles/tailwind.css

```diff
@@ -61,4 +61,14 @@ progress[value]::-moz-progress-bar {
   .break-word {
     word-break: break-word;
   }
+
+  /* This global setting ensures that all animated SVGs are paused if a user prefers reduced motion.
+  Media queries that are directly added to SVG styles are lost when the SVG is imported as a react component */
+  @media (prefers-reduced-motion: reduce) {
```

Comment:
Can we share this with `fxa-react` instead of needing it two places?

Reply:
I'd initially done that and it didn't work as expected. Maybe something wasn't working right with my build. I can retry.

Reply:
Still running into some issues when added to `fxa-react` - but I believe we only need in in fxa-settings. I've kept a single copy in this file and removed the one in `fxa-react`

Reply:
One thing I thought about later... you'd mentioned before that we had to use `prefers-reduced-motion` inside the SVG I think due to CSP. Why does this work now?

Reply:
I don't recall there being an issue with CSP, but the approach I had initially tried last time to globally apply the preference hadn't worked. This time around, SVGO was stripping out the media queries from the svg, so this approach turned out to be a more efficient way to apply the preference (hadn't thought of it before).